docs/advanced/Concurrency.md

```diff
@@ -4,11 +4,14 @@ In the basics section, we saw how to use the helper effects `takeEvery` and `tak
 
 In this section we'll see how those helpers could be implemented using the low-level Effects.
 
+These helpers return a forked task, which creates an attached task that will block execution of the saga that invokes the helper. If you need the helpers to create a detached task, which is what the official redux-saga's implementation of `takeLatest` and `takeEvery` does, then use `spawn` instead. 
+
```

Comment:
in my root saga, if I use the implemntation of takeLatest or takeEvery that uses fork, then it blocks the other yield statements.

```
export default function* rootSaga () {
    yield takeLatest();
    yield takeLatest();// this line will not execute unless spawn() is used
}
```